components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/GatewayArtifactsMgtDAO.java

```diff
@@ -77,23 +79,25 @@ public void addGatewayPublishedAPIDetails(String APIId, String APIName, String v
      * @param streamLength - Length of the stream
      * @throws APIManagementException if an error occurs
      */
-    public void addGatewayPublishedAPIArtifacts(String APIId, String gatewayLabel, ByteArrayInputStream bais,
+    public boolean addGatewayPublishedAPIArtifacts(String APIId, String gatewayLabel, ByteArrayInputStream bais,
```

Comment:
Even though the function signature has been changed to return a boolean, no action is being done by checking the return type at call time.

Reply:
We are checking the return flag in the test method https://github.com/wso2/carbon-apimgt/pull/9002/files#diff-5d5216ef9cc19eb82608ade2690638cdR113